awx/main/analytics/collectors.py

```diff
@@ -158,15 +158,17 @@ def instance_info(since):
     instances = models.Instance.objects.values_list('hostname').annotate().values(
         'uuid', 'version', 'capacity', 'cpu', 'memory', 'managed_by_policy', 'hostname', 'last_isolated_check', 'enabled')
     for instance in instances:
-        info = {'uuid': instance['uuid'],
-                'version': instance['version'],
-                'capacity': instance['capacity'],
-                'cpu': instance['cpu'],
-                'memory': instance['memory'],
-                'managed_by_policy': instance['managed_by_policy'],
-                'last_isolated_check': instance['last_isolated_check'],
-                'enabled': instance['enabled']
-                }
+        instance_info = {
+            'uuid': instance['uuid'],
+            'version': instance['version'],
+            'capacity': instance['capacity'],
+            'cpu': instance['cpu'],
```

Comment:
which metric is used for k8s based installs? mCPU?

Reply:
Don't know. But this PR is merely using the collectors we didn't change anything about how or what is collected. If we need to make a change to the collectors, we would open a ticket for that. This diff is just from me fixing the formatting because it was awful before.